templates/donor-funds-thanks.html.twig

```diff
@@ -7,10 +7,16 @@
 
     <p>
         Thank you for transferring {{ transferAmount }} into your Big Give account.
-        {{ newBalance }} is now available for you to donate to your chosen charities in our campaign. Please note that
```

Comment:
removing all mention of newBalance because it's likely to change immediately after this email is sent.